### ISSUE TITLE
NULL string interpreted as empty string

### ISSUE THREAD

**Issue by bbkr** [CurtTilmes/perl6-dbmysql#4]
To reproduce:

```sql
CREATE TABLE foo ( bar char(1) ) engine=InnoDB;
INSERT INTO foo (bar) VALUES (NULL);
```

```perl6
$mysql.execute("SELECT bar FROM foo").value.perl.say;
```

Result (should be `Any()`):
```
""
```

Same applies to VARCHAR type.

**Comment by CurtTilmes** [comment]
Thanks for testing!

Fixed typed NULL handling with commit 288dc45eee410bda5a6bc0c41b062a73212c565d

**Comment by bbkr** [comment]
Thanks for super-quick fix! Works great.

BTW: Returning undefined specific types instead of Any:U is great idea. One advantage I've immediately found is that it allows to pass data from database straight to Perl functions with narrow signatures making the code more compact.

Thanks again for your time invested in this module.
